services/hanke-service/src/main/resources/db/changelog/changesets/086-change-car-lane-nuisance-enum.sql

```diff
@@ -0,0 +1,134 @@
+--liquibase formatted sql
+--changeset Teemu Hiltunen:086-change-car-lane-nuisance-enum
+--comment: Change 'hankealue.kaistahaitta' enum values and 'tormaystarkastelutulos.kaistapituushaitta' and 'applications.applicationData.areas[n].tyoalueet[m].tormaystarkastelutulos.kaistapituushaitta' index values
+
+-- update hankealue.kaistahaitta enum values
+ALTER TABLE hankealue
+    ADD COLUMN kaistahaitta_temp VARCHAR;
```

Comment:
I don't think the temp column does anything. The values can be updated in place.